suite-native/atoms/src/FullAlertBox/FullAlertBox.tsx

```diff
@@ -0,0 +1,78 @@
+import { Icon } from '@suite-native/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from '../Box';
+import { Button } from '../Button/Button';
+import { HStack, VStack } from '../Stack';
+import { Text } from '../Text';
+import { variantToColorMap, variantToIconName } from './presets';
+import { FullAlertProps, FullAlertStyles } from './types';
+
+const containerStyle = prepareNativeStyle<Pick<FullAlertStyles, 'backgroundColor' | 'borderColor'>>(
+    (utils, { backgroundColor, borderColor }) => ({
+        backgroundColor: utils.colors[backgroundColor],
+        borderWidth: utils.borders.widths.small,
+        borderColor: utils.colors[borderColor],
+        borderRadius: utils.borders.radii.r12,
+        padding: utils.spacings.sp16,
+    }),
+);
+
+export const FullAlertBox = ({
+    variant = 'neutral',
+    title,
+    description,
+    primaryButtonLabel,
+    onPressPrimaryButton,
+    onPressSecondaryButton,
+    secondaryButtonLabel,
+    primaryButtonProps,
+    secondaryButtonProps,
```

Comment:
nit: optional props should come as last.

```suggestion
    title,
    description,
    primaryButtonLabel,
    onPressPrimaryButton,
    onPressSecondaryButton,
    secondaryButtonLabel,
    primaryButtonProps,
    secondaryButtonProps,
    variant = 'neutral',
```

Reply:
have no objections here but may I ask why? as it is an object the order does not really matter. Will change tho

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/a78326105b817d33005372141527efac30a93050